assets/src/styles/blocks/Covers/TakeActionCovers.scss

```diff
@@ -146,12 +146,17 @@
 
     .btn {
       display: block;
-      background-color: var(--cover-card-btn-background-color, $orange);
-      border-color: var(--cover-card-btn-border-color, $orange);
+
+      --block-covers--card-btn-- {
```

Comment:
Applying the syntax here meant changing the variable name. Did this in a few places, but only after checking that the previous name wasn't used anywhere else (e.g. https://github.com/search?q=org%3Agreenpeace+%22--cover-card-btn-border-color%22&type=code).